packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/FloatFixedEnum.cs

```diff
@@ -0,0 +1,13 @@
+// <auto-generated/>
+
+#nullable disable
+
+namespace UnbrandedTypeSpec.Models
+{
+    public enum FloatFixedEnum
+    {
+        One,
```

Comment:
Fixed enums are generated in this way:
if its value is integers, we put them in the definition, such as:
```csharp
public enum IntFixedEnum
{
	A =1, B = 2, C = 100
}
```
and it only has deserialization, in its serialization, we just use `(int)value`.
For other values, we just do this:
```csharp
public enum FloatFixedEnum
{ A, B, C }
```
and its extension class has methods both for serialization and deserialization.

Reply:
ah so the spec called `One` therefore we write it as one.  Can we change the spec to use `OnePointOne` or `OneDotOne` just to make it clear in the test projects what we are looking at.

Reply:
Ok if they did not assign names in the spec, I am pretty sure we will generate something like `_1`.
I will add something in the spec so that we have an example